basilisk/tests/test_prompt_attachments_panel.py

```
@@ -0,0 +1,669 @@
+
+    def test_add_very_long_filename(self, panel):
+        with tempfile.TemporaryDirectory() as d:
+            fn = "a"*255 + ".txt"
+            path = Path(d)/fn
+            try:
+                path.write_text("x")
+                panel.add_attachment(str(path))
+            except OSError:
+                pass
+
+    def test_add_attachment_insufficient_permissions(self, panel):
```

Ruff (0.11.9)

Missing docstring in public method

(D102)